### ISSUE TITLE
Non-synced patterns missing focus outline in pattern category inserter

### ISSUE THREAD

**Issue by jeryj** [WordPress/gutenberg#60872]
- Open main inserter in header
- Go to Patterns Tab
- Select a pattern category
- Tab to patterns
- Use arrow keys to move between patterns
- Focus outline is not visible for non-synced patterns. They should have a black focus ring like is present when hovering the items.

<img width="643" alt="Standard pattern selected with tooltip showing without any focus ring" src="https://github.com/WordPress/gutenberg/assets/967608/ffe2e804-6a96-4754-a6af-b2ffb3a355a0">

**Comment by jeryj** [comment]
Duplicate of https://github.com/WordPress/gutenberg/issues/60211
